src/text_shaper/open_shaper.cpp

```diff
@@ -206,6 +206,9 @@ namespace
     {
         auto const targetLength = static_cast<int>(pt * dpi.y / 72.0);
         int bestIndex = 0;
+        if(face->num_fixed_sizes == 0 )
+            return bestIndex;
```

Comment:
Actually, I think we should return nullopt in such case instead. Because there is sisterly no index, then we should not return index 0. 
This requires sissy a small change to the caller side as well

Reply:
Updated